src/App.tsx

```diff
@@ -1,11 +1,42 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+
+import { DisplayCard } from "./components/DisplayCard/DisplayCard";
+import { fetchSingleCharacter } from "./utils/fetchFunctions";
+
 const App = () => {
-	return (
-		<>
-			<div>
-				<h1>Pokémon Battle</h1>
-			</div>
-		</>
-	);
+  const [nameInputValue, setNameInputValue] = useState<string>("");
+  const [entities, setEntities] = useState<any[]>([]);
+
+  const handleNameInput = (event: ChangeEvent<HTMLInputElement>) => {
+    setNameInputValue(event.target.value);
```

Comment:
Not a big deal here and in this context it doesn't matter. But probably the majority of the time here you want to reference `event.currentTarget` instead of `event.target`. 

The difference being `event.target` references the element that triggered the event, and `event.currentTarget` references the element that is listening to the event. 

There are plenty of scenarios where you want to use `event.target` while doing event delegation to know which specific element triggered the event, but in most cases like this, you, your event listener is on the input field, if for some reason a child element were to trigger a change event it would bubble up to this, and you would be referencing the wrong element.


Again not worth changing here because it doesn't make a difference in this scenario, but just wanted to throw that information out there.

Reply:
I did wonder what the difference was between the two, thank you!